src/mlbstandings/web.py

```
@@ -3,25 +3,35 @@
         (requests.exceptions.HTTPError, urllib.error.HTTPError, requests.exceptions.ConnectionError, requests.exceptions.Timeout),
         max_time=600,
-        giveup=lambda e : isinstance(e, requests.exceptions.HTTPError) and e.response.status_code not in set([429, 500, 503]),
+        giveup=lambda e: Web.giveup(e),
         max_value=60
     )
     def read(url: str) -> str:
-        print(f'Web.read({url}')
-        if url.find('https://www.baseball-reference.com/') == 0:
+        if url.startswith('https://www.baseball-reference.com/') == 0:
+            print(f'Using urllib in Web.read({url})')
             req = urllib.request.Request(url)
```

Ruff (0.11.9)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)